scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/TarWriter.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.diging.scriptoocloud.core.service.impl;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import org.apache.commons.compress.archivers.tar.TarArchiveEntry;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+
+public class TarWriter {
+
+    private FileOutputStream out;
+    private TarArchiveOutputStream tOut;    
+    private int len;
```

Comment:
needs a better name. what lengths is is? resulting tar file, number of entries, etc.